docs/index.md

```diff
@@ -3,28 +3,34 @@ myst:
   html_meta:
     "description": "The Volto user interface (UI) is a React-based frontend for the Plone CMS. It is the default user interface starting with the release of Plone 6."
     "property=og:description": "The Volto user interface (UI) is a React-based frontend for the Plone CMS. It is the default user interface starting with the release of Plone 6."
-    "property=og:title": "Volto UI"
-    "keywords": "Volto, Plone, frontend, user interface, React"
+    "property=og:title": "Seven"
+    "keywords": "Seven, Plone, frontend, user interface, React"
 ---
 
 (volto-index-label)=
 
-# Volto UI
+# Seven
 
-The Volto user interface (UI) is a React-based frontend for the [Plone content management system](https://plone.org).
-It is the default user interface starting with the release of Plone 6.
+The Seven user interface (UI) is a React-based frontend for the [Plone content management system](https://plone.org).
```

Comment:
I'd entertain adjectives such as "fast", "elegant", or "intuitive". Reason being to promote its strengths, instead of creating a pissing match between frontends, although Volto already won it.

```suggestion
The Seven user interface (UI) is a React-based frontend for [Plone](https://plone.org).
```

Reply:
All temptative things, of course, just to have something in here.